stdlib/scripts/check-docstrings.py

```diff
@@ -0,0 +1,33 @@
+# ===----------------------------------------------------------------------=== #
+# Copyright (c) 2024, Modular Inc. All rights reserved.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions:
+# https://llvm.org/LICENSE.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===----------------------------------------------------------------------=== #
+
+import subprocess
+from pathlib import Path
+import sys
+
+# TODO: Use the "mojo doc" directly when there is an option to
+# fail if warnings are present (something like -Werror for gcc).
+
+def main():
+    # This is actually faster than running "mojo doc" on each file since 
```

Comment:
**Suggestion** I don't know if this help text is available externally, but `mojo doc` is taking a <package_directory> here as its argument, and `stdlib/src` is said package directory.  The intent is for `mojo doc` to work at a full (public) `mojopkg` level and not a file level.  In the JSON emitted internally from the `mojo` tool, it includes the warnings and metadata for each file in the package.

Perhaps clarify that in the comment here to avoid reader confusion/how others would incorporate `mojo doc` into their own libraries?

Reply:
This comment will never be displayed to a constributor, it will only be seen if one looks at the source code. Do you think that's necessary to be this detailed in this case? Furthermore this will is temporary we'll just be waiting for https://github.com/modularml/mojo/issues/2447 to be fixed